### PR TITLE
Add com.github.GradienceTeam.Gradience

### PR DESCRIPTION
**Pull request by LyesSaadi**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub.
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by LyesSaadi** [comment]
Still reading App Maintenance. Sorry for the old erroneous PRs :P!

**Comment by 0xMRTT** [comment]
Thanks 👍🚀

File: pypi-dependencies.json [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+        {
+            "name": "python3-regex",
+            "buildsystem": "simple",
+            "build-commands": [
+                "pip3 install --verbose --exists-action=i --no-index --find-links=\"file://${PWD}\" --prefix=${FLATPAK_DEST} \"regex\" --no-build-isolation"
+            ],
+            "sources": [
+                {
+                    "type": "file",
+                    "url": "https://files.pythonhosted.org/packages/f7/e1/8b46ff54516faea4ea9469c4f5b50f2e9e73131f54d19e6827141698a822/regex-2022.8.17.tar.gz",
```
Comment (hfiguiere) [inline pypi-dependencies.json:29]: this one is already in the next module.

File: pypi-dependencies.json [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+                    "sha256": "75e636fd3e0fb872693f23ccb8a5ff2cd578801251f3a4f6854c6a5d437d3c04"
+                },
+                {
+                    "type": "file",
+                    "url": "https://files.pythonhosted.org/packages/a6/d0/bdee3e8a7ba5f2dc8bacea10f4fdd1264b1dd9e53d96d318773180634a00/reportlab-3.6.11.tar.gz",
+                    "sha256": "04fc4420f0548815d0623e031c86a1f7f3f3003e699d9af7148742e2d72b024a"
+                },
+                {
+                    "type": "file",
+                    "url": "https://files.pythonhosted.org/packages/70/bb/7a2c7b4f8f434aa1ee801704bf08f1e53d7b5feba3d5313ab17003477808/lxml-4.9.1.tar.gz",
```
Comment (hfiguiere) [inline pypi-dependencies.json:96]: you build this one by hand too.

**Review by hfiguiere — CHANGES_REQUESTED** [review]

**Comment by hfiguiere** [comment]
bot, build com.github.GradienceTeam.Gradience

**Comment by LyesSaadi** [comment]
This won't build, 0.2.0 isn't out yet, but will be in a few hours. We submitted this a bit early :sweat_smile:!

File: pypi-dependencies.json (hunk 1 above)
Comment (LyesSaadi) [reply pypi-dependencies.json:29]: Oh yes! This file is generated automatically. It picked up a redundant dependency. Will get it out of Gradience.

File: pypi-dependencies.json (hunk 2 above)
Comment (LyesSaadi) [reply pypi-dependencies.json:96]: Apparently, @0xMRTT has troubles when not building it by hands. I will investigate that later, but will get this out of `pypi-dependencies.json` until I figure it out.

**Comment by 0xMRTT** [comment]
Published 🚀

**Comment by LyesSaadi** [comment]
Yeah, if the build can be restarted, it should now work.

File: pypi-dependencies.json (hunk 2 above)
Comment (LyesSaadi) [reply pypi-dependencies.json]: I removed one of the duplicates until I investigate it.

Comment (hfiguiere) [reply pypi-dependencies.json]: the know problem is that it is in the Sdk, but not the runtime, so it gets skipped but then things fail.

I have complained a few times about that and got told it's by design.

**Comment by LyesSaadi** [comment]
Btw, @hfiguiere, adding co-maintainers/changing maintainers is as easy as giving access to the GitHub repo right? I made the PR because @0xMRTT was unable to, but I think it's more legitimate for him to be the main maintainer, and me as a co-maintainer.

**Comment by hfiguiere** [comment]
You can be both added, that's definitely not a problem, quite the opposite.

**Comment by 0xMRTT** [comment]
Thanks 👍🚀

**Pull request closed without merging**